code/game/objects/items/cprbot.dm

```diff
@@ -0,0 +1,113 @@
+/obj/item/cprbot_item
+	name = "CPRbot"
+	desc = "A compact CPRbot 9000 asemply"
+	icon = 'icons/obj/structures/machinery/aibots.dmi'
+	icon_state = "cprbot"
+	w_class = SIZE_MEDIUM
+	var/deployment_path = /obj/structure/machinery/bot/cprbot
+
+/obj/item/cprbot_item/attack_self(mob/user as mob)
+	if (..())
+		return TRUE
+
+	if(user)
+		deploy_cprbot(user, user.loc)
+
+/obj/item/cprbot_item/proc/deploy_cprbot(mob/user, atom/location)
+	if(!user || !location)
+		return
+
+	if (istype(user))
+		if(!skillcheck(user, SKILL_MEDICAL, SKILL_MEDICAL_MEDIC))
+			to_chat(user, SPAN_WARNING("You don't seem to know how to use [src]..."))
+			return
+
+	qdel(src)
+
+	// Proceed with the CPRbot deployment
+	var/obj/structure/machinery/bot/cprbot/entity = new deployment_path(location)
+	if(entity)
+		entity.add_fingerprint(user)
+		entity.owner = user
+
+/obj/item/cprbot_item/afterattack(atom/target, mob/user, proximity)
+	if(proximity && isturf(target))
+		var/turf/T = target
```

Comment:
Replace `T` with a more descriptive name to make it easier to understand what it represents at a glance